composition_stats/__init__.py

```diff
@@ -449,6 +449,9 @@ def clr(mat):
        rows = compositions and
        columns = components
        each composition (row) must add up to unity (see :ref:`closure()`)
+    ignore_zero : bool
+        whether to ignore zeros in ``mat``. This reproduces the behavior of the
+        ``compositions`` R package.
```

Comment:
I think it might be better to put the note about the R package in the main description, instead of the parameter text.

Also, this should probably be indented with 3 spaces, to match the weird indentation of everything else.

```suggestion
    ignore_zero : bool, optional
       whether to ignore zeros in ``mat`` (default: false)
```